report/sla.go

```diff
@@ -404,3 +404,38 @@ func SLASummary(probers []probe.Prober) string {
 	summary += "\n" + global.FooterString()
 	return summary
 }
+
+// SLACSVSection set the CSV format for SLA
+func SLACSVSection(r *probe.Result) string {
+	text := "%s, %s, up(%s), down(%s), %.2f%%, %d(%s), %s, %s, %s"
+	return fmt.Sprintf(text, r.Name, r.Endpoint,
```

Comment:
prefer ? 
```
return fmt.Sprint(
```

over 

```
return fmt.Sprintf(
```

Reply:
here, using Sprintf instead of Sprint is better. and using Sprintf way can be easy to maintain.